fuzzers/libafl/builder.Dockerfile

```diff
@@ -26,29 +26,23 @@ RUN apt-get update && \
     apt-get remove -y llvm-10 && \
     apt-get install -y \
         build-essential \
-        llvm-11 \
-        clang-12 \
-        cargo && \
+        lsb-release wget software-properties-common gnupg && \
     apt-get install -y wget libstdc++5 libtool-bin automake flex bison \
         libglib2.0-dev libpixman-1-dev python3-setuptools unzip \
         apt-utils apt-transport-https ca-certificates joe curl && \
+    wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && ./llvm.sh 16 \
```

Comment:
oh I think this will not enable llvm 16..
(it will make /usr/bin/clang-16 etc.)

but the issue is iirc the base image has llvm-15 installed (not from apt) and /usr/bin/clang is symlinked to it

Reply:
you can do like what I did here
https://github.com/tokatoka/fuzzbench/blob/cmplog_ctx_experiment/fuzzers/libafl_cmplog_ctx/builder.Dockerfile#L37

Reply:
The script fails pretty instantly..